house/constants.py

```diff
@@ -1,2 +1,4 @@
 HOME_PAGE_ROUTE = 'home.html'
+HOUSE_PAGE_ROUTE = 'house_view/mine_page.html'
```

Comment:
When you add new lines to a file, you should add them at the end of the file, to avoid conflicts. (Also when you resolve the conflicts locally, when you rebase, add your new lines at the end of the file)

This would make the conflict resolution process easier for you when you rebase 😄